build/posix.sh

```diff
@@ -107,6 +107,8 @@ CURL="curl --silent --location --retry 3 --retry-max-time 30"
 if [ "$DARWIN" = true ]; then
   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
     | sh -s -- -y --no-modify-path --profile minimal
+  rustup default nightly
+  export RUSTFLAGS="-Zlocation-detail=none -Zfmt-debug=none"
```

Comment:
The `RUSTFLAGS` should be appended here to ensure these flags are not overwritten:
https://github.com/lovell/sharp-libvips/blob/77fb93cce8beb4989e3de37b9f707cb205946a83/build/posix.sh#L95-L97
```suggestion
  export RUSTFLAGS+=" -Zlocation-detail=none -Zfmt-debug=none"
```

Though, we could also use `export CARGO_PROFILE_RELEASE_TRIM_PATHS="all"` there when building with Rust nightly, see:
https://github.com/kleisauke/wasm-vips/commit/9da512b2d2bbe6361a355c00c966f2e214b20a72#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R177